spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -1016,9 +1016,15 @@
     let(:expected_community_collection_string_center_for_file_id) { "science-in-society-scientific-images-contest::2018 Scientific Images Contest Winners" }
     let(:map_file_to_prism_community_collections_irrc) { described_class.new(generic_file_with_community_collection_match, collection_store.data) }
 
+    let!(:collection_pnb) { make_collection(user, title: "Pediatric Neurology Briefs: Volume 05 Issue 07", id: "pnb-5-7", member_ids: [generic_file_pnb.id]) }
```

Comment:
A bit of rearrangement and renaming to hopefully make the testing pattern more clear.

Here you can also see an example of what I mean by a collection id without an issue number in case that was unclear